relay-server/src/services/outcome.rs

```diff
@@ -210,7 +215,20 @@ impl Outcome {
                 code_opt.as_ref().map(|code| Cow::Borrowed(code.as_str()))
             }
             #[cfg(feature = "processing")]
-            Outcome::CardinalityLimited => None,
+            Outcome::CardinalityLimited(ids) => {
+                use std::fmt::Write;
+                let mut buf = String::new();
+                write!(&mut buf, "CardinalityLimited:").unwrap();
```

Comment:
No prefix needed here, the outcome id already contains the information. The prefix exists for others because multiple different reasons share the same outcome, but here we just have cardinality limits.